src/node_crypto.cc

```diff
@@ -3952,6 +3954,37 @@ void SignBase::CheckThrow(SignBase::Error error) {
   }
 }
 
+int SignBase::GetRSAOptions(Environment* env, v8::Local<v8::Object> options,
```

Comment:
@bnoordhuis fwiw, I’m good with the current approach. It’s readable (imho) and microperformance probably isn’t too relevant when compared against the cost of the crypto operations itself.